Sources_v3/StreamChatUI/AppearanceSetting.swift

```diff
@@ -11,10 +11,6 @@ public protocol AppearanceSetting: AnyObject {
     func defaultAppearance()
 }
 
-public extension AppearanceSetting {
-    func defaultAppearance() { /* default empty implementation */ }
-}
-
```

Comment:
Same problem with `setUpLayout` happened with this too...

Reply:
Yep